Assets/StreamingAssets/Data/Furniture.xml

```diff
@@ -671,4 +670,25 @@
         <LocalizationCode>furn_rt_generator</LocalizationCode>
         <UnlocalizedDescription>furn_rt_generator_desc</UnlocalizedDescription>
     </Furniture>
+
+    <Furniture type="astro_wall">
+    <Name>Rock Wall</Name>
+    <Description>This is a wall of rock!</Description>
+    <TypeTag>Wall</TypeTag>
+    <TypeTag>Non-buildable</TypeTag>
+    <TypeTag>Non-deconstructible</TypeTag>
+    <MovementCost>0</MovementCost>
+    <Width>1</Width>
+    <Height>1</Height>
+    <LinksToNeighbours>true</LinksToNeighbours>
+    <EnclosesRooms>true</EnclosesRooms>
+    
+    <Params>
+     <Param name="ore_type" value="None" />
+    </Params>
+
```

Comment:
@TheRiven If you don't mind then I'll go ahead and add it since you have allow maintainer pushes enabled.